ci-automation/vendor-testing/equinix_metal.sh

```diff
@@ -28,6 +28,15 @@ BASE_URL="http://${BUILDCACHE_SERVER}/images/${CIA_ARCH}/${CIA_VERNUM}"
 run_kola_tests() {
     local instance_type="${1}"; shift
     local instance_tapfile="${1}"; shift
+    local timeout_args=()
+
+    if [[ "${instance_type}" =~ \.xlarge\. ]]; then
+        # let's double timeouts for xlarge instances
+        timeout_args+=(
+            --equinixmetal-launch-timeout=20m
+            --equinixmetal-install-timeout=1h30m
```

Comment:
I think the launch is more likely to be the one hitting the timeout, or? Maybe just double that.

Reply:
Launch timeout is 10 minutes by default, so here it's already doubled.